h2o-py/h2o/expr.py

```diff
@@ -343,7 +343,7 @@ def _tabulate(self, tablefmt="simple", rollups=False):
                 domain = v['domain']  # Map to cat strings as needed
                 x = ["" if math.isnan(idx) else domain[int(idx)] for idx in x]
             elif t == "time":
-                x = ["" if math.isnan(z) else time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(z / 1000)) for z in x]
+                x = ["" if math.isnan(z) else time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(z / 1000)) for z in x]
```

Comment:
I am ok with that...it can be just confusing for end-users...

Just an idea: can we provide an optional parameter to force timezone setting in this call? I know it is not the best option...but still better than global variable.